openapi.yaml

```diff
@@ -692,7 +692,261 @@ paths:
                                  example:
                                     available: 107079929856
                                  description: The available storage on the device in bytes
-
+   /organizations/{orgId}/configurations/{configurationId}:
+      get:
+         summary: Get configuration
+         description: Get a configuration by id.
+         parameters:
+            - name: orgId
+              in: path
+              required: true
+              schema:
+                 type: string
+              description: The ID of the organization. Locate the orgId under Organization Settings > API Keys.
+            - name: configurationId
+              in: path
+              required: true
+              schema:
+                 type: string
+              description: The ID of the configuration.
+         responses:
+            '200':
+               description: A configuration object
+               content:
+                  application/json:
+                     schema:
+                        type: object
+                        properties:
+                              id:
+                                 type: string
+                                 example: 6qwM6FzJE4o708iqTpWc
+                                 description: The ID of the configuration on ManageXR.
+                              name:
+                                 type: string
+                                 example: Trial
+                                 description: The name given to the configuration on ManageXR.
+                              registrationCode:
+                                 type: string
+                                 example: f7GbHvH
+                                 description: The registration code of the configuration.
+                              deviceCount:
+                                 type: number
+                                 example: 15
+                                 description: Number of devices that use the configuration.
+                              deviceExperience:
+                                 type: string
+                                 enum:
+                                    - HOME_SCREEN
+                                    - KIOSK
+                                    - DEFAULT
+                                 description: Device experience set on the configuration.
+                              apps:
+                                 type: object
+                                 properties:
+                                    app-id:
```

Comment:
This (and similar pattern for other assets) is a workaround to both
1. convey the fact that `apps` is a map that maps an app id to the app object
2. document the structure of each app object

The official OpenAPI syntax for this is 

```
apps:
  type: object
  additionalProperties:
     type: object
     properties:
       // define the structure of an app object here
```

More on `additionalProperties` here: https://swagger.io/docs/specification/data-models/dictionaries/.

However, mintlify can't seem to parse `additionalProperties` field. I've searched up their doc but they didn't mention this.

Reply:
As discussed, we're updating the endpoint to return arrays rather than object maps. Resolving this!